src/pages/Home/OpenChannelModal.tsx

```diff
@@ -67,10 +69,14 @@ const OpenChannelModal: FC<Props> = ({ onClose }) => {
     setAmount(+event.target.value);
   };
 
+  const convertedBalance = balance ? convertMSatToSat(balance) : 0;
+
   return createPortal(
     <ModalDialog close={onClose}>
-      <h2 className="mb-2 text-lg font-bold">{t("home.open_channel")}</h2>
-      <article className="py-8">
+      <h2 className="text-lg font-bold">{t("home.open_channel")}</h2>
+      <article className="my-5">
+        <AvailableBalance balance={convertedBalance!} />
```

Comment:
Does TS need the `!`? If yes, is there a way to avoid the `!`?

Reply:
In this case yes, because `convertMSatToSat` could also return `null`.

Is not relevant in this case, thus I forced it with `!`

Reply:
I feel like ! should be the last option. But not a blocker.

Reply:
I can create an issue for it if you like

There are some more `!` in the code anyway